qiskit/transpiler/passes/scheduling/calibration_creators.py

```diff
@@ -174,7 +175,11 @@ def get_calibration(self, params: List, qubits: List) -> Schedule:
             QiskitError: if the control and target qubits cannot be identified or the backend
                 does not support cx between the qubits.
         """
-        theta = params[0]
+        if isinstance(params[0], ParameterExpression):
+            theta = float(params[0]._symbol_expr)
```

Comment:
`float(params[0])` should accomplish the same (and will raise a `TypeError` if the `ParameterExpression` has unbound parameters).